ocaml/xapi-idl/cluster/cluster_interface.ml

```diff
@@ -15,12 +15,45 @@ type debug_info = string [@@deriving rpcty]
 (** Name of the cluster *)
 type cluster_name = string [@@deriving rpcty]
 
-(** An IPv4 address (a.b.c.d) *)
-type address = IPv4 of string [@@deriving rpcty]
+type ipv4v6 = IPv4 of string | IPv6 of string [@@deriving rpcty]
```

Comment:
Could  we simplify `ipv4v6` to `ip` since there are only two version v4 or v6 mostly used?